recipes-mender/mender-artifact/mender-artifact.bb

```diff
@@ -0,0 +1,22 @@
+DESCRIPTION = "Mender artifact information"
+HOMEPAGE = "https://mender.io"
+LICENSE = "Apache-2.0"
+
+inherit allarch
+
+PV = "0.1"
+
+do_compile() {
+    echo "# populate this file with build info" > ${B}/artifact-info
+}
+
+do_install() {
+    install -d ${D}${sysconfdir}/mender
+    install -t ${D}${sysconfdir}/mender ${B}/artifact-info
+    ln -s artifact-info ${D}${sysconfdir}/mender/build_mender
```

Comment:
I'm thinking we should get rid of this symlink while we can. I don't think anyone external is relying on it. Makes sense?